editor/scene/texture/texture_region_editor_plugin.cpp

```diff
@@ -463,13 +452,7 @@ void TextureRegionEditor::_texture_overlay_input(const Ref<InputEvent> &p_input)
 					edited_margin = -1;
 				} else {
 					undo_redo->create_action(TTR("Set Region Rect"));
-					if (node_sprite_2d) {
-						undo_redo->add_do_method(node_sprite_2d, "set_region_rect", node_sprite_2d->get_region_rect());
-						undo_redo->add_undo_method(node_sprite_2d, "set_region_rect", rect_prev);
-					} else if (node_sprite_3d) {
-						undo_redo->add_do_method(node_sprite_3d, "set_region_rect", node_sprite_3d->get_region_rect());
-						undo_redo->add_undo_method(node_sprite_3d, "set_region_rect", rect_prev);
-					} else if (node_ninepatch) {
+					if (node_ninepatch) {
```

Comment:
In these if statements, put Control-based things first, and Node2D/Node3D things after it. This is arguably more organized, but the motivation here was so you can put `#ifndef _2D_DISABLED` above the `} else if (condition) {`.